tensorflow_addons/optimizers/optimizer_test_base.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Base class for optimizer tests."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+
+
+class OptimizerTestBase(tf.test.TestCase):
+    """Base class for optimizer tests.
+  
+  Optimizer tests may inherit from this class and define test functions
```

Comment:
Could you fix doc style (align them)? `make code-format` might help you https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md#development-environment